src/Command/RunCommand.php

```diff
@@ -78,20 +78,21 @@ protected function configure(): void
                     'Name of the Test framework to use ("%s")',
                     implode('", "', TestFrameworkTypes::TYPES)
                 ),
-                ''
+                Container::DEFAULT_TEST_FRAMEWORK
             )
             ->addOption(
                 'test-framework-options',
                 null,
                 InputOption::VALUE_REQUIRED,
-                'Options to be passed to the test framework'
+                'Options to be passed to the test framework',
+                Container::DEFAULT_TEST_FRAMEWORK_EXTRA_OPTIONS
```

Comment:
Why do we need defaut values in the input, where they are `null`s? Looks strange to me and in order to understand what is the value - we need to find a value of this const, which is `null`, meaning "there is no default value"

Reply:
Clarified some bits in https://github.com/infection/infection/pull/1256/files#r410943399, "we need to find a value of this const," is precisely what I'm interested in since changing the default value of the option can result in the default being different than the container known default. Having this here is a nudge to double check that container default